frontend/src/store/modules/geo_location/actions.js

```diff
@@ -1,11 +1,37 @@
-import {CHANGE_SELECTED_PERIOD, CHANGE_SELECTED_PARAMETER} from "./types/actions";
-import {SET_SELECTED_PERIOD, SET_SELECTED_PARAMETER} from "./types/mutations";
+import {
+    CHANGE_SELECTED_PERIOD,
+    CHANGE_SELECTED_PARAMETER,
+    FETCH_GEO_LOCATION_ITEMS
+} from "./types/actions";
+import {
+    SET_SELECTED_PERIOD,
+    SET_SELECTED_PARAMETER,
+    SET_GEO_LOCATION_ITEMS,
+    SET_IS_FETCHING,
+    RESET_IS_FETCHING
+} from "./types/mutations";
+import {getGeoLocationItems} from '@/api/geo_location/geoLocationItemsService';
+import {getTimeByPeriod} from '@/services/periodService';
 
 export default {
     [CHANGE_SELECTED_PERIOD]: (context, payload) => {
         context.commit(SET_SELECTED_PERIOD, payload.value);
+        context.commit(SET_IS_FETCHING);
+        context.dispatch(FETCH_GEO_LOCATION_ITEMS);
     },
     [CHANGE_SELECTED_PARAMETER]: (context, parameter) => {
         context.commit(SET_SELECTED_PARAMETER, parameter);
+    },
+    [FETCH_GEO_LOCATION_ITEMS]: context => {
+        const period = getTimeByPeriod(context.state.selectedPeriod);
+
+        period.startDate = period.startDate.unix();
```

Comment:
it's better to define or variable or call `period.startDate.unix()` directly to `getGeoLocationItems` params. Since modifying Object, that defined as const not quite good Idea. Also keep in mind that any object in JS passing by reference

It also applies to next row